.travis.yml

```diff
@@ -65,8 +65,10 @@ jobs:
 
     # Test against PHP 8
     - php: nightly
+      env: SYMFONY_DEPRECATIONS_HELPER=weak
       before_install:
         - composer config platform.php 7.4.99
+        - composer remove --dev doctrine/orm
```

Comment:
You could either use `--no-update` or remove the `update` line below since it is superfluous.

Reply:
You're right! Done.